src/main/java/com/github/tommyettinger/digital/MathTools.java

```diff
@@ -53,6 +53,18 @@ private MathTools() {
      * false-positive equivalence with very small inputs.
      */
     public static final float FLOAT_ROUNDING_ERROR = 0x1p-20f; // was 0.000001f
+    
+    /**
+     * The smallest measurable difference between a given {@code float} value and a directly adjacent value.
+     * Useful for converting a 64-bit {@code long} value to a gradient between 0 and 1.
```

Comment:
Also a bit of a nit-pick here, this is useful for converting non-negative up-to-24-bit `int`s or `long`s to fit in a uniformly-spaced 0 to 1 range. It can work on more than just `long`, and it probably doesn't do what you want if you multiply an arbitrary (possibly negative) `long` by this.